src/core/mount.c

```diff
@@ -116,16 +134,33 @@ static bool mount_is_bind(const MountParameters *p) {
         return false;
 }
 
-static bool mount_is_bound_to_device(const Mount *m) {
+static bool mount_is_bound_to_device(Mount *m) {
```

Comment:
Losing `const` here is a bit unfortunate.
Maybe change get_mount_parameters() into a macro (MOUNT_PARAMETERS?). This will allow us to constify a bunch callers.

Reply:
Quite frankly I don't see the big benefit of const. i.e. macros are ugly beasts, and whether to have a const on some internal, static function somewhere sounds like a tiny benefit to me.